Fix segfault in caffe2 slice test

supriyar opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#31801 Fix segfault in caffe2 slice test**

Summary: Try to fix issue #30764

Test Plan: python test/onnx/test_utility_funs.py TestUtilityFuns

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D19315046](https://our.internmc.facebook.com/intern/diff/D19315046)

supriyar committed:
Fix segfault in caffe2 slice test
Summary: Try to fix 30764

Test Plan: python test/onnx/test_utility_funs.py TestUtilityFuns

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit d73b66ff:


**None of the build failures appear to be your fault**.

* **1/1** [broken upstream](https://dr.pytorch.org/master-timeline.html?count=1&sha1=9e9bfbfd8d893cdf8205db67f220551214787d7f&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35) at merge base 9e9bfbfd since Jan 02
  <details>
  <summary>
  You may want to rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
  ```
  git fetch origin viable/strict
  git rebase --onto viable/strict $(git merge-base origin/master HEAD)
  ```
  If your commit is older than `viable/strict`:
  ```
  git fetch origin viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=d73b66ff259c6f3cddd1724854c5f89a9ed29bf5).

### :construction: 1 upstream failure recognized by patterns:
These builds matched patterns, but were probably **caused by upstream breakages**:
* [pytorch_macos_10_13_py3_test](https://dr.pytorch.org/build-details.html?build_id=79580248) from Jan 02



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

houseroad reviewed (approved):
Good catch.

supriyar committed:
Update on "Fix segfault in caffe2 slice test"


Summary: Try to fix issue #30764

Test Plan: python test/onnx/test_utility_funs.py TestUtilityFuns

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D19315046](https://our.internmc.facebook.com/intern/diff/D19315046)

[ghstack-poisoned]
